R/utility.R

```diff
@@ -173,6 +173,10 @@ getJobResult <- function(jobId = "", ...){
     results <- downloadBlob(jobId, paste0("result/", jobId, "-merge-result.rds"))
```

Comment:
How long will this downloadBlob operation take? Should we have a progress bar on it? Perhaps just a verbose command that says something like "Getting job results ..."